symphony/lib/toolkit/class.emailgateway.php

```diff
@@ -610,13 +631,15 @@ protected function getSectionAttachments()
                      . $this->contentInfoString($file['mime-type'], $file['file'], $file['filename'], $file['charset'])
                      . EmailHelper::base64ContentTransferEncode($file_content);
             } else {
-                if (!$tmp_file === false) {
-                    $filename = $original_filename;
-                } else {
-                    $filename = $file['file'];
+                if ($this->_ignore_attachment_errors === false) {
+                    if (!$tmp_file === false) {
```

Comment:
I don't think that PSR has any guidelines for this, at least I haven't found anything. :-(

I will do what you proposed.